docs/3.0/deploy/static-infrastructure-examples/docker.mdx

```diff
@@ -0,0 +1,189 @@
+---
+title: Serve flows in a long-lived Docker container
+description: Learn how to serve a flow in a long-lived Docker container
+---
+
+The `.serve` method allows you to easily elevate a flow to a deployment, immediately executing scheduled work [as a local process](/3.0/deploy/run-flows-in-local-processes).
```

Comment:
'immediately executing scheduled work' seems like a bit of an oxymoron. I suggested an alternate wording, but feel free to do something else.
```suggestion
The `.serve` method allows you to schedule runs of your flow and execute those runs [in a local process](/3.0/deploy/run-flows-in-local-processes).
```